layouts/shortcodes/related_posts.html

```diff
@@ -0,0 +1,9 @@
+
+Other posts about barcodes on my blog:
```

Comment:
"barcodes" needs to be replaced by a template placeholder for the tag.

Reply:
sorry . I forgot to update this